src/editor tweaks.js

```diff
@@ -0,0 +1,451 @@
+/*
+👣
+@file editor tweaks
+@summary Various adjustments to the editor that are too minor for individual plugins.  Each tweak is togglable.
+@license MIT
+@author Violgamba (Jon Heard)
+
+@description
+Various editor tweaks, or adjustments to the editor that are too minor for individual plugins.
+
+This plugin's event contains a togglable text field for each tweak.  The field defaults to "true",
+and can be set to "false" to disable the tweak.
+NOTE: togglable text fields were chosen over tag fields for easier toggling.
+
+Below are a list of the tweaks, starting with their togglable field.  Note that this list is subject
+to change as new tweaks are added to this plugin.
+
+TWEAKS LIST
+- tweak--new-plugins-have-filename-tag - Each imported plugin's fields will begin with a tag named
+	after the filename the plugin was loaded from.  This lets one quickly identify plugin events.
+- tweak--new-fields-default-to-tag-type - New fields begin as "tag" types, rather than "text" types.
+	This is useful for users who add tag events more frequently than text events.
+- tweak--tileset-export-ordered-by-tile - When the tileset is exported, the graphics are re-ordered
+	to match the tile order set in the editor.  This lets the user define the export order and makes
+	sure that all animation frames are consecutive.
+- tweak--save-button-shows-when-saved - When the game is saved, the save button dims until a change
+	occurs, wherein it turns back to yellow.  If the change is undone, the save button dims again.
+	This lets one know when the editor is in a saved state, removing the need for redundant save
+	clicks to be sure.  NOTE: the button can still be clicked while dimmed.
+- tweak--field-name-column-maximized - This widens the name column of the fields list ui.  Useful if
+	using long field names.
+- tweak--editing-hotkeys - 'delete', 'ctrl-c', 'ctrl-v' & 'ctrl-x' work as expected on events,
+	fields & tiles right after clicking them.  Arrow keys move events, fields & tiles around right
+	after clicking them.  'ctrl-shift-z' triggers REDO, rather than UNDO.
```

Comment:
this would probably also be worth submitting to bipsi itself

Reply:
See above.  This change is significantly more involved than the prior one and, thus, I suspect it would take longer for it to make it into the main application.